pyproject.toml

```diff
@@ -1,11 +1,190 @@
-[build-system]
+[project]
+name = "stingray"
+dynamic = [
+    "version"
+]
+description = "Time Series Methods For Astronomical X-ray Data"
+readme = "README.rst"
+authors = [
+    { name = "Stingray Developers", email = "spectraltiming-stingray@googlegroups.com" }
+]
+license = { text = "MIT" }
+requires-python = ">=3.8"
```

Comment:
@masonng-astro @taldcroft I added all these metadata to the pyproject.toml file, including the supported Python versions and a contact (a Google Group which is admittedly underused but an easy way to contact us).
The python version encoded here is picked up by the badge in the README